index.html

```diff
@@ -11,21 +12,116 @@
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
+    <![endif]-->
 </head>
 
 <body>
 
-    <div class="container">
-      <div class="row">
-          <div class="col-sm-12">
-              <h1>Start Here!</h1>
+  <div class="container pt-3 pb-3">
+    <div class="jumbotron mt-2">
+      <img class="img-fluid mb-2" src="../img/banner.jpg" alt="">
+      <h1>Jared Cuffe
+        <small>Software Developer</small>
+      </h1>
+      <p class="mb-0">
+        <a href="mailto:jcuffe@gmail.com">jcuffe@gmail.com</a>
+        <br /><a href="http://github.com/jcuffe">github.com/jcuffe</a>
+        <br />916-764-4943</p>
+    </div>
+    <div class="row justify-content-center">
+      <div class="btn-group mt-1 mb-4">
+        <button id="skills" class="btn btn-custom active-custom">Skills</button>
+        <button id="experience" class="btn btn-custom">Experience</button>
+        <button id="references" class="btn btn-custom">References</button>
+      </div>
+    </div>
+    <section class="skills">
+      <h3 class="mb-4">Skills</h3>
+      <div class="row ml-1 mr-1">
+        <div class="col-sm-4">
+          <div class="card">
+            <div class="card-body">
```

Comment:
If you have the extra time, you can add your media query for the phone view. There is some weird spacing in regards to card body as you hit the phone view.